### ISSUE TITLE
[a11y] loading indicator should have correct semantics role

### ISSUE THREAD

**Issue by chunhtai** [flutter/flutter#161631]
When using https://api.flutter.dev/flutter/material/CircularProgressIndicator-class.html, the loading spinner should have following role
| OS | role |
|--------|--------|
| web |- |
| ios | - |
| macos | NSAccessibilityProgressIndicatorRole |
| windows | - |
| android | - | 

When using https://api.flutter.dev/flutter/material/LinearProgressIndicator-class.html, the linear indicator should have following role
| OS | role |
|--------|--------|
| web |progressbar |
| ios | - |
| macos | NSAccessibilityLevelIndicatorRole |
| windows | ROLE_SYSTEM_PROGRESSBAR |
| android | - |

**Comment by rkishan516** [comment]
@Piinks  So, what will be next steps?

**Comment by hannah-hyj** [comment]
this issue should be closed by the reland: https://github.com/flutter/flutter/pull/178402
